packages/rum-core/src/domain/view/viewCollection.spec.ts

```diff
@@ -228,4 +231,23 @@ describe('viewCollection', () => {
 
     expect(rawRumViewEvent.display?.scroll).toBeUndefined()
   })
+
+  it('should include configuration.start_session_replay_recording_manually value', () => {
+    let { lifeCycle, rawRumEvents } = setupBuilder
+      .withConfiguration({ startSessionReplayRecordingManually: false })
+      .build()
+    lifeCycle.notify(LifeCycleEventType.VIEW_UPDATED, VIEW)
+    expect(
+      (rawRumEvents[rawRumEvents.length - 1].rawRumEvent as RawRumViewEvent)._dd.configuration
+        .start_session_replay_recording_manually
+    ).toBe(false)
+    ;({ lifeCycle, rawRumEvents } = setupBuilder
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** having a blank line could been nice to separate the two cases. I understand that it's a prettier issue, but maye add a comment or something